### PR TITLE
Added a few operators to the Velocity class.

### PR DESCRIPTION
**Pull request by pylaligand**
#1968

### CODE REVIEW

File: packages/flutter/lib/src/gestures/velocity_tracker.dart [hunk 1]
```diff
@@ -225,6 +225,12 @@ class Velocity {
     return pixelsPerSecond == typedOther.pixelsPerSecond;
   }
 
+  Velocity operator -() => new Velocity(pixelsPerSecond: -pixelsPerSecond);
+  Velocity operator -(Velocity other) =>
+      new Velocity(pixelsPerSecond: pixelsPerSecond - other.pixelsPerSecond);
+  Velocity operator +(Velocity other) =>
+      new Velocity(pixelsPerSecond: pixelsPerSecond + other.pixelsPerSecond);
+
```
Comment (Hixie) [inline packages/flutter/lib/src/gestures/velocity_tracker.dart]: Please don't line break if you use `=>`, use `{ return ... }` instead.

Comment (Hixie) [inline packages/flutter/lib/src/gestures/velocity_tracker.dart]: also, please put these above the operator ==, so that == and hashCode are together.

**Comment by Hixie** [comment]
LGTM.

FWIW, our style guide is here:
https://github.com/flutter/engine/blob/master/sky/specs/style-guide.md
https://github.com/flutter/engine/blob/master/sky/specs/design.md

File: packages/flutter/lib/src/gestures/velocity_tracker.dart (hunk 1 above)
Comment (pylaligand) [inline packages/flutter/lib/src/gestures/velocity_tracker.dart]: All done.

**Pull request merged**